mifosng-android/src/main/java/com/mifos/mifosxdroid/uihelpers/MFDatePicker.java

```diff
@@ -21,6 +21,10 @@
  */
 public class MFDatePicker extends DialogFragment implements DatePickerDialog.OnDateSetListener {
 
+    private static final String DATE_KEY = "date_present";
+    private static final int set_max_date = 0;
```

Comment:
Don't use 0 and 1. Either use Strings in Caps like `FOR_MIN_DATE` or use ENUM. 0 and 1 isn't easy to comprehend for the new user.